domain-models-maven-plugin/src/test/java/org/folio/rest/tools/ClientGeneratorTest.java

```diff
@@ -12,7 +12,7 @@
 
 public class ClientGeneratorTest {
 
-  private static final Pattern TRAILING_SPACE_PATTERN = Pattern.compile("\\s+$", Pattern.MULTILINE);
+  private static final Pattern TRAILING_SPACE_PATTERN = Pattern.compile("\\s+", Pattern.MULTILINE);
```

Comment:
No need to ignore _all_ whitespace.
Simply temporarily disable `@After cleanUp()` and copy the generated
`target/generated-sources/raml-jaxrs/org/folio/rest/client/TestResourceClient.java` to
`src/test/resources/clients/TestClient.txt`.